### PR TITLE
Cleanup and usability improvements

### PR DESCRIPTION
**Pull request by nearmax**
* Script for publishing crates;
* Removed unnecessary lazy_static;
* Remove env_test altogether, now that we have a workaround for cargo bug;
* Removing check_all.sh which fails now for no apparent reason, even though build succeeds;
* Bumping version.

### CODE REVIEW

File: ci-test.sh [hunk 1]
```diff
@@ -25,8 +25,6 @@ else
     fi
 
     # Only testing it for one configuation to avoid running the same tests twice
-    echo "Checking compilation of wasm32 for all examples"
-    ./examples/check_all.sh
```
Comment (evgenykuzyakov) [inline ci-test.sh]: It was actually helpful cause it was building wasm32 target. We should instead investigate why it fails to build now?

Comment (nearmax) [reply ci-test.sh]: ./build_all.sh is building it anyway.

Comment (evgenykuzyakov) [reply ci-test.sh]: But `build_all.sh` is not part of CI

Comment (nearmax) [reply ci-test.sh]: Fixed.

**Pull request merged**